src/poliastro/spheroid_location.py

```diff
@@ -19,18 +19,17 @@ def __init__(self, lon, lat, h, a, c):
         h: ~astropy.units.quantity.Quantity
             geodetic height
 
-        a: ~astropy.units.quantity.Quantity
-            polar semi-major axis
+        body: ~poliastro.bodies.Body
+            Planetary body the spheroid location lies on
 
-        c: ~astropy.units.quantity.Quantity
-            equatorial semi-minor axis
         """
         self._lon = lon
         self._lat = lat
         self._h = h
-        self._a = a
-        self._b = a
-        self._c = c
+        self._body = body
```

Comment:
Do we need to store the particular `Body`?

Reply:
You're right, no need for adding it now. I was thinking about adding relevant methods later, but we can add it back again if and when it's needed